packages/webapp/pages/account/security.tsx

```diff
@@ -58,9 +38,14 @@ const AccountSecurityPage = (): ReactElement => {
   } = usePrivilegedSession();
 
   const { mutateAsync: resetPassword } = useMutation(
-    (params: ValidateResetPassword) => submitKratosFlow(params),
+    ({
+      onPasswordReset: _,
```

Comment:
Why the _ deconstruct here?

Reply:
Just so we don't pass the parameter to the actual endpoint, and so the linting won't shout of the variable being unused.

Reply:
Seems like we don't need the `_` and lint doesn't seem to react to it.